type Error = Box<dyn std::error::Error>;

fn main() -> Result<(), Error> { return Ok(()); }
